packages/design-system/src/components/label.tsx

```diff
@@ -0,0 +1,78 @@
+import type { LabelHTMLAttributes, ReactNode, Ref } from "react";
+import { forwardRef } from "react";
+import { styled } from "../stitches.config";
+import { theme } from "../stitches.config";
+
+const StyledLabel = styled("label", {
+  boxSizing: "border-box",
+  flexShrink: 0,
+
+  // shared style
+  py: 2,
+  border: "1px solid transparent",
+  borderRadius: theme.borderRadius[3],
+  fontSize: theme.fontSize[3],
```

Comment:
This should come from typography token. In Figma you can see which token to use like this:

<img width="468" alt="Screenshot 2023-01-20 at 18 07 26" src="https://user-images.githubusercontent.com/825702/213716482-90e8e369-aa72-4457-8d71-425c39f9eb54.png">

In this case it's "Labels - Sentence Case" which will correspond to `typography.labelsSentenceCase` once https://github.com/webstudio-is/webstudio-designer/pull/821 is merged

Reply:
Can leave as is with a todo if you're going to merge earlier. Saying just so you know this is coming

Reply:
Will rebase once you merge